### ISSUE TITLE
Stamina damage is still absurd.

### ISSUE THREAD

**Issue by QuietlyWhisper** [space-wizards/space-station-14#21558]
There's realistically no counter to this... we can sit here nerfing stun baton, stamina numbers... whatever, but this isnt engaging in the slightest for the recieving player. Either stun functionality need to be overhauled or armor needs to have high resistance to stamina based damage. 

Theres absolutely no reason why non lethals should be the prefered takedown method for heavily armed terrorists. Let non lethals be used for civilians and use lethals on actual killers to give both sides engaging gameplay that lasts longer than a few seconds. Why should non lethals simply entirely bypass all armor and plans? Its no wonder antagonists resort to the easiest metagamy methods of crippling the station when anyone on the crew already do to them!

https://cdn.discordapp.com/attachments/557322456013209647/1172267006729400380/godsbestnukie.mp4?ex=655fb1ba&is=654d3cba&hm=3ef0a578197588669595cde6236cad27fae8803d126ec148510aa91d7a7a5de3&

**Comment by asperger-sind** [comment]
mfw cdn downloads the video instead of playingit, here's a link to general on discord where it embeds
https://discord.com/channels/310555209753690112/675078881425752124/1172405307817861211

**Comment by LankLTE** [comment]
I'm not really in the mood to have this discussion for the billionth time, but I do want to add that I think rubbers specifically are super overtuned (in reference to the clip above).

**Comment by Brandon-Huu** [comment]
MFW billion dollar soldier is defeated by rubber balls

**Comment by asperger-sind** [comment]
this one is gonna be long so if you want to avoid tactics and strategy and gamesense skip to some place i highlighted
also holy shit an actual real scenario as opposed to theoreticals on theoreticals



Looking at the video they got shot like 9 times during the first "cycle", looking at the .yml for baseBulletRubber it deals 22 stamina damage, so 5 connected hits is enough to stamcrit anyone, the detective after that kept shooting them so I count 13 shots total from what I assume is an mk58.

The nukie was also most likely injured before or during the firefight as exampled in the moment where the nukie uses both a freedom and an EMP implant despite being on low HP surrounded by 4 players, essentially being marked for death.

From a tactical and strategical perspective this fight is even more of a mess. The nukie in question went into 3 armed secoffs without being on stims or meth, while having the duffelbag reducing his speed; speed is key to fighting secoffs or armed crew of any kind.

The nukie then went through a 1x1 hallway that expands into what could hold at least 3 people shooting at the same time with little-to-no crossfire.
All of this while going in with an e-sword and e-shield both of which aren't exactly ideal for fighting guns while not being on your terms.

The nukie could be described as an "attacker" side here as they were going into sec as opposed to having sec go into them, this means they can't effectively use corners or relocate because they also use a melee build requiring them to be in melee range to trade damage, neither can they cause crossfire or bait shots, the nukie also avoided fleeing from a 4:1 fight despite not being prepared for one, I'm assuming they had atmos access, but they could also go through the disposal in atmos, or emag into space.

This isn't all but I don't want to go too much into theoreticals or assumptions for this clip, the nukie did not play well, they picked a terrible fight on terrible conditions and got stomped on the way very fast by rubbers, to which I'll get later in more specific and relevancy to the discussion.

Sec played this one well enough, they hit clean shots, avoided crossfire, kept as a group and had possible backup options and plans, did they play well enough to be rewarded with a nukie kill? Perhaps, assuming the first 5 shots are all that matters because after that the nukie goes into stamcrit they might've been rewarded too greatly on a gun balance level, because guns who can "kill" a nukie with an eshield and esword in 5 or less shots are very few and should stay few.

**VVV part to skip to VVV**

Sec was rewarded for their good gameplay by getting a nukie kill, which is fine, but what isn't fine is the sample window for good gameplay, five shots in the span of two seconds to decide the outcome of a fight with what is commonly described as a dangerous and prepared foe is insanely high TTK which is also only applied on one party, as sec doesn't get punished so harshly for missing a clip, especially in group.

I personally think that high TTK sucks, it's not very exciting to play with or engaging to play against, getting killed by 5 rubbers is very high TTK considering the fire-rate of a pistol, even despite the nukie playing poorly in the above scenario they get punished too harshly and rubbers can of course still do the same provided both parties play well and one lands 5 bullets.

What do I propose? Make armor resistant to rubbers, it makes logical sense, it makes for gameplay sense, you want to stun someone in military grade armor? Shoot them with a disabler which is worse than rubbers or go into close quarters and trade hits, high risk high reward, or opt in for more consistent damage with lethals.

**Comment by Just-a-Unity-Dev** [comment]
stun damage discussion number 848283334

**Comment by asperger-sind** [comment]
> stun damage discussion number 848283334

rain you dont get it we have an actual real practical scenario to go off this shit is a gold mine

**Comment by Just-a-Unity-Dev** [comment]
> > stun damage discussion number 848283334
> 
> rain you dont get it we have an actual real practical scenario to go off this shit is a gold mine

someone said this on the last stun damage discussion

**Comment by deltanedas** [comment]
top secret military grade hardsuit vs 5 high speed rubber ball who would win

**Comment by liltenhead** [comment]
Yeah as it stands right now. If you want to go loud as a syndi, you need to preemptively make or steal stun recovery drugs. Or potentially sabotage gravity since it's rather hard to cuff someone who is sliding around. Or the most popular option is sit in space with a jetpack and slowly space everything. There's more to this but it really does force players into really specific often cheesy strats like grille camping and such.

Having armor isn't very meaningful since most of the guns have 1 second to stun "ttk" or even faster than that. Common tactics for this as well is to top load rubber bullets, since if you lose half your stamina you are slowed massively making it much easier to land lethal shots, or just go for the full stun and cuff. And even if you don't get cuffed, you'll still drop whatever is in your hand. 

Getting hit by one beanbag slows your movement speed by more than half for 3 seconds. And landing any 3 rubber bullets will also do the same.

Most guns won't even crit a naked person with lethal bullets before you can stam crit with rubber bullets. And since the only way to increase stamina is a very short duration stim, there's no real effective way to deal with rubber rounds. Funnily enough the two dedicated items for stuns, (the baton and disabler) have the slowest stun speeds in the game.

**Comment by CrigCrag** [comment]
>gets shot with 5 metal bullets
>owie, good thing i have armor
>gets shot with 5 rubber bullets
>keels over in pain, physically unable stand up, has a heart attack, eyes pop out of head, entire family tree feels the pain of a thousand suns

**Comment by EmoGarbage404** [comment]
Just based off of the gameplay in the vid, asperger-sind had a pretty appropriate overview of it. The nukie played like garbage and got ganged up on. 

I do agree that, based on the numbers, the stamina damage per bullet is quite high. I'm not opposed to lowering them a bit. Of course, it should be easier to stamcrit than to kill, or else shooting an armored target with rubber bullets is going to kill them before the stamina is depleted.

But as a condemnation of the stamina combat balance as a whole? This feels kinda weak.

**Comment by Mudsson** [comment]
The clear solution is to just give regular bullets the same Stam damage as rubber bullets (joke)

I don't see why the time to stun and time to kill with bullets cant be the same by making the Stam damage equal to the pierce damage then letting armor block both. Rubber bullets are still good against unarmored targets and identical to regular bullets for time to stun or kill, only you you'd rather kill then stun.

I suppose the issue with that is the slowdown at half stamina, but I'm no expert on this. It still seems silly the response to heavily armored targets is rubber over lead.

**Comment by IProduceWidgets** [comment]
So I'm gonna just go on a tangent here, but...

We could have material "hardness" or "conductiveness" that determines what damage set a weapon deals.

Armor is harder, and stops all but the most penetrative weapons (think bullets, fire axes, etc). things that don't penetrate could then do lesser damage perhaps of a different type. (Think shot with bullet = piercing, shot with rubber = blunt).

A hard suit is both hard, and insulating, so it would not conduct an electric disabler round for instance.

**Comment by lexs0sa** [comment]
the best anti-nukie gun is mapped into maints, been an issue since beanbags existed.

**Comment by kontakt** [comment]
Stale. Balance discussion should be handled by relevant workgroups.
